node/_crypto/crypto_browserify/evp_bytes_to_key_test.ts

```diff
@@ -10,7 +10,6 @@ Deno.test("salt buffer length is 7", function () {
     function () {
       EVP_BytesToKey(Buffer.alloc(5), Buffer.alloc(7), 1, 1);
     },
-    undefined,
```

Comment:
I think this should be `Error`. Now we use `assertThrows(fn: () => unknown, msg?: string)` signature here and the 2nd arg is not compared with the thrown error message (it's used as just informative `msg`)